code/modules/games/cards.dm

```diff
@@ -8,12 +9,15 @@
 	icon = 'icons/obj/playing_cards.dmi'
 	var/list/cards = list()
 	var/cooldown = 0 // to prevent spam shuffle
+	var/decktype = "playingcards" /// Should match parentdeck for any related card packs.
```

Comment:
If you make the cards use subtypes per deck type (e.g., `datum/playingcard/cag` versus `datum/playingcard/cardemon`), then you can use a type here, not have to track strings (like reagents, which is a really icky part of that system), and use istype() checks